examples/guru1d1.cpp

```diff
@@ -42,43 +38,47 @@ int main(int argc, char *argv[])
   } else            // or, NULL here means use default opts...
     finufft_makeplan(type, dim, Ns, +1, ntransf, tol, &plan, NULL);
 
+  const std::complex<double> I(0.0, 1.0);
+
   // generate some random nonuniform points
-  vector<double> x(M);
+  std::vector<double> x(M);
   for (int j = 0; j < M; ++j)
-    x[j] = PI * (2 * ((double)rand() / RAND_MAX) - 1); // uniform random in [-pi,pi)
+    x[j] = PI * (2 * ((double)std::rand() / RAND_MAX) - 1); // uniform random in [-pi,pi)
   // note FINUFFT doesn't use std::vector types, so we need to make a pointer...
   finufft_setpts(plan, M, x.data(), NULL, NULL, 0, NULL, NULL, NULL);
 
   // generate some complex strengths
-  vector<complex<double>> c(M);
+  std::vector<std::complex<double>> c(M);
   for (int j = 0; j < M; ++j)
-    c[j] =
-        2 * ((double)rand() / RAND_MAX) - 1 + 1i * (2 * ((double)rand() / RAND_MAX) - 1);
+    c[j] = 2 * ((double)std::rand() / RAND_MAX) - 1 +
+           std::complex<double>(0.0, 1.0) *
```

Comment:
Can we use ``I`` which was defined above for a reason?    I must say this ``std::`` everywhere is highly unpleasant!   I'm not sure why in our example codes we're killing ``using namespace std;``   ....